### ISSUE TITLE
HTTP Error 502 (Bad Gateway) on retrieve_sra_metadata.py

### ISSUE THREAD

**Issue by bentsherman** [SystemsGenetics/GEMmaker#126]
I get this HTTP error when I try to run the ath26k experiment on the NRP:
```
  Downloading run IDs   7300 -   7399 of  26055...
  Downloading run IDs   7400 -   7499 of  26055...
  Downloading run IDs   7500 -   7599 of  26055...
  Downloading run IDs   7600 -   7699 of  26055...
  Downloading run IDs   7700 -   7799 of  26055...
  Downloading run IDs   7800 -   7899 of  26055...
  Downloading run IDs   7900 -   7999 of  26055...
  Downloading run IDs   8000 -   8099 of  26055...
  Downloading run IDs   8100 -   8199 of  26055...
  Downloading run IDs   8200 -   8299 of  26055...
  Downloading run IDs   8300 -   8399 of  26055...
  Downloading run IDs   8400 -   8499 of  26055...
  Downloading run IDs   8500 -   8599 of  26055...
  Downloading run IDs   8600 -   8699 of  26055...
  Downloading run IDs   8700 -   8799 of  26055...
  Downloading run IDs   8800 -   8899 of  26055...
  Downloading run IDs   8900 -   8999 of  26055...
  Downloading run IDs   9000 -   9099 of  26055...
  Downloading run IDs   9100 -   9199 of  26055...
  Downloading run IDs   9200 -   9299 of  26055...
  Downloading run IDs   9300 -   9399 of  26055...
  Downloading run IDs   9400 -   9499 of  26055...
  Downloading run IDs   9500 -   9599 of  26055...
  Downloading run IDs   9600 -   9699 of  26055...
  Downloading run IDs   9700 -   9799 of  26055...
  Downloading run IDs   9800 -   9899 of  26055...
  Downloading run IDs   9900 -   9999 of  26055...
  Downloading run IDs  10000 -  10099 of  26055...
  Downloading run IDs  10100 -  10199 of  26055...
  Downloading run IDs  10200 -  10299 of  26055...
  Downloading run IDs  10300 -  10399 of  26055...
  Downloading run IDs  10400 -  10499 of  26055...
  Traceback (most recent call last):
    File "/workspace/projects/systemsgenetics/GEMmaker/bin/retrieve_sra_metadata.py", line 263, in <module>
      download_runs_meta(run_ids, page_size=args.PAGE_SIZE)
    File "/workspace/projects/systemsgenetics/GEMmaker/bin/retrieve_sra_metadata.py", line 50, in download_runs_meta
      response_obj = urllib.request.urlopen(request)
    File "/usr/lib/python3.5/urllib/request.py", line 163, in urlopen
      return opener.open(url, data, timeout)
    File "/usr/lib/python3.5/urllib/request.py", line 472, in open
      response = meth(req, response)
    File "/usr/lib/python3.5/urllib/request.py", line 582, in http_response
      'http', request, response, code, msg, hdrs)
    File "/usr/lib/python3.5/urllib/request.py", line 510, in error
      return self._call_chain(*args)
    File "/usr/lib/python3.5/urllib/request.py", line 444, in _call_chain
      result = func(*args)
    File "/usr/lib/python3.5/urllib/request.py", line 590, in http_error_default
      raise HTTPError(req.full_url, code, msg, hdrs, fp)
  urllib.error.HTTPError: HTTP Error 502: Bad Gateway
```

It doesn't happen on Palmetto so I suspect the cause could be specific to NRP. Obviously this issue is related to #94 but I think the causes are different. A bad request (error 400) indicates something wrong with the request body, such as what @spficklin suggested in that issue. However a bad gateway error would indicate an error happened on the server or one of it's proxies.

Just posting this here for documentation purposes. I will try to decrease the page size from 1000 to 100 so that the `retrieve_sra_metadata.py` script is less likely to overload the server that is fetching the metadata.

**Comment by bentsherman** [comment]
Whoops, just realized that the page size is already 100 by default... I guess I'll decrease it to 50?

**Comment by spficklin** [comment]
@bentsherman is this the only error you're getting?  I thought in the meeting today you had said there was another error message you sometimes get.

**Comment by bentsherman** [comment]
@spficklin I think this error is the main one that I get on the NRP now. It's the one where I think a proxy server is failing somewhere. I'd like to leave this issue open but we need to keep debugging on the NRP, maybe get Dima and the other NRP admins to help, in order to know how to fix this issue.

**Comment by spficklin** [comment]
I think this issue is resolved as per the PR #204
